rooms.js

```diff
@@ -163,6 +165,95 @@ var Room = (function () {
 		}
 		return true;
 	};
+	//mute handling
+	Room.prototype.runMuteTimer = function () {
+		if (this.muteTimer) return;
+		if (this.mutedUsers[0].time < Date.now() + 5) {
+			// this prevents lag from bugging the timer
+			this.mutedUsers[0].time = Date.now() + 5;
+		}
+		var self = this;
+		this.muteTimer = setTimeout(function () {
+			var user = Users(self.mutedUsers[0].userid);
```

Comment:
Do you mean `runMuteTimer` is the function you want to check if it needs to run the mute timer? I'm not sure I understand what you're wanting now.